pkgs/development/compilers/llvm/10/default.nix

```diff
@@ -155,6 +155,7 @@ let
       extraBuildCommands = ''
         echo "-rtlib=compiler-rt -Wno-unused-command-line-argument" >> $out/nix-support/cc-cflags
         echo "-B${targetLlvmLibraries.compiler-rt}/lib" >> $out/nix-support/cc-cflags
+        echo "-L${targetLlvmLibraries.libunwind}/lib" >> $out/nix-support/cc-ldflags
```

Comment:
Given that the library is conditional on !isWasm below and targetPlatform.useLLVM, this should probably be along side that.

Reply:
`clang` links against libunwind when `-unwindlib=libunwind` is set. Hence if conditioning on `targetPlatform.useLLVM`, `llvmPackages.clangUseLLVM` still cannot find libunwind.

Reply:
It's definitely curious that we gate `-lunwind` on `targetPlatform.useLLVM` but not `--unwindlib=libunwind`.

Tracing back to [where the `-lunwind` was added](https://github.com/NixOS/nixpkgs/commit/37194a325de090747ce5f97a38de4329708c1912#diff-5c97508ac7912c7688e27a77c3ddca9f3c8e5a5fc448d79736561b771ca64cebR125-R126) doesn't reveal any obvious reason for the discrepancy and I'm actually not clear on why the `-lunwind` was needed in the first place (my understanding is that [it's implied](https://github.com/llvm/llvm-project/blob/a78f5c064975f4d597ed1b18206d579748eb91cb/clang/lib/Driver/ToolChains/CommonArgs.cpp#L1887)). Perhaps there were scenarios where the linker was being invoked directly with the expectation that libunwind would be pulled in?

Regardless, doesn't need to block this PR. Adding this `-L` in shouldn't hurt.